temporian/implementation/numpy/operators/test/moving_count_test.py

```diff
@@ -54,6 +53,34 @@ def test_cc_wo_sampling(self):
             _i32([1, 2, 3, 4, 1]),
         )
 
+    def test_cc_window_0(self):
+        assert_array_equal(
+            operators_cc.moving_count(
+                _f64([1, 2, 3, 5, 20]),
+                0,
+            ),
+            _i32([0, 0, 0, 0, 0]),
+        )

```

Comment:
Is this behavior intended? Should a 0-sized window not include the item it ends on? It behaves like this with both fixed and variable window

Reply:
I think it's the expected behavior. In fact, this is only possible calling directly to the cpp operator, because null durations are not allowed from the python api.
We have `tp.duration.shortest` for cases where you want to make sure that a window doesn't overlap two different timestamps.

Reply:
Gotcha, fixed 👍🏼 Thanks!